### ISSUE TITLE
[BUG] twinx zoom works only for the second graph

### ISSUE THREAD

**Issue by JoaoAparicio** [JuliaPlots/Plots.jl#2906]
## Details

When using twinx, the drag and drop zoom only zooms the second graph

### Backends

This bug occurs on ( insert `x` below )

Backend      | yes | no  | untested
-------------|-----|-----|---------
gr (default) |     |     |x
pyplot       |    |     |x
plotly       |   x |     |
plotlyjs     |     |     |x
pgfplotsx    |     |     |x
inspectdr    |     |     |x

### Versions

Plots.jl version:
Backend  version (`]st -m`): `[91a5bcdd] Plots v1.5.5`
Output of `versioninfo()`:
```
Julia Version 1.4.2
Commit 44fa15b150* (2020-05-23 18:35 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, haswell)
Environment:
  JULIA_EDITOR = atom  -a
  JULIA_NUM_THREADS = 4
```

**Comment by JoaoAparicio** [comment]
Would someone care to confirm that they can replicate?

**Comment by daschw** [comment]
Yes, I can confirm. However, I doubt that this can be fixed in Plots easily.
